app/Http/Requests/StoreInventory.php

```diff
@@ -27,16 +26,8 @@ public function authorize()
     public function rules()
     {
         return $this->clean([
-            'entity_id' => 'required|exists:entities,id',
             'item_id' => 'nullable|array|required_without:name',
             'item_id.*' => 'exists:items,id',
-            'name' => 'nullable|string|required_without:item_id',
```

Comment:
The rules is now just 2 entities (and clean is useless now since that's only used for updates). This isn't how you do object oriented extending :)